### DISCUSSION TITLE
Reac_Fuel Chemical Formula

### DISCUSSION THREAD

**Discussion by glevyils** [firemodels/fds#12949] [Q&A]
When importing an SFPE reaction (PMMA, Polyurethane, etc), PyroSim automatically creates the REAC_FUEL species. Most properties of the species are undefined - leading to a warning message that I've read in other discussions can sometimes/mostly be ignored. 

However, the chemical formula for the REAC_FUEL is always input as C1.0H1.8O0.17N0.17, regardless of the reaction or fuel type. Is this an intentional default, or should the chemical formula be edited depending on the fuel source? Is there a reason that the REAC_FUEL does not autopopulate with values from SFPE Handbook Tables?

Snip of PMMA formula and Molecular weight from Table A.31 (page 3446) compared to snip of autopopulated values in Pyrosim

![image](https://github.com/firemodels/fds/assets/170022298/410fccaa-cde3-4395-9946-cfde20e195ca)

![image](https://github.com/firemodels/fds/assets/170022298/ebd0e1c5-96bc-4357-af36-8d6643434863)

**Comment by drjfloyd** [comment]
This forum does not provide support for Pyrosim. Please contact Thunderhead Engineering.
